kytos/core/controller.py

```diff
@@ -790,26 +789,33 @@ def unload_napps(self):
         for (username, napp_name) in list(self.napps.keys()):  # noqa
             self.unload_napp(username, napp_name)
 
-    def reload_napp(self, username, napp_name):
-        """Reload a NApp."""
-        self.unload_napp(username, napp_name)
-
-        mod_name = '.'.join(['napps', username, napp_name, 'main'])
+    def reload_napp_module(self, username, napp_name, napp_file):
+        """Reload a NApp Module."""
+        mod_name = '.'.join(['napps', username, napp_name, napp_file])
```

Comment:
This is a big change to be included together with linter fixes. Please separate this in another branch / PR.